pstatus-graphql-ktor/src/main/kotlin/gov/cdc/ocio/processingstatusapi/models/query/PageSummary.kt

```diff
@@ -24,5 +24,11 @@ data class PageSummary(
     var pageSize: Int = 0,
 
     @GraphQLDescription("Total number of items that can be provided")
-    var totalItems: Int = 0
+    var totalItems: Int = 0,
+
+    @GraphQLDescription("List of the senderIds")
+    var senderIds: List<String> = listOf(),
+
+    @GraphQLDescription("List of the jurisdictions")
```

Comment:
Change the description to be more informative.  Something like, "List of all the jurisdictions in the entire dataset matching the filter criteria, not just this page."

Reply:
Updated according to the suggestions